utils/date.tsx

```diff
@@ -9,6 +9,17 @@ export const fixMalformedISO8601DateString = (date: string) => (MISSING_TIMEZONE
 // The National Avalanche Center API expects 'YYYY-MM-DD' date-strings in query parameters, and it operates in UTC.
 export const apiDateString = (date: Date) => formatInTimeZone(date, 'UTC', 'yyyy-MM-dd');
 
+// Forecasts expire in the middle of a calendar day, so the forecast that we expect to be valid at any given time during
+// the day changes based on the relationship of that time to the expected expiry time, as recorded for the avalanche
+// center.
```

Comment:
I think this comment could be improved by just adding a concrete example or two, something like 

requestedTime | expiryTimeHours | returns
--- | --- | ---
datetime before expiry hours | 18 | result
datetime after expiry hours | 18 | result

Reply:
also, a couple of tests in `date.test.tsx` would probably help clarify this